### ISSUE TITLE
[BUG/ISSUE] RRTMG timestep is not copied from GCHP.rc to Input_Geos%TS_RAD

### ISSUE THREAD

**Issue by yantosca** [geoschem/gchp#279]
### Description of the problem

I was doing a GCHP integration test on the fullchem_RRTMG simulation. It failed with this error (which is apparently due to some incorrect config file setup):
```config
TIMESTEPS SETTINGS
------------------
Chemistry  Timestep [sec]   :   1200
Convection Timestep [sec]   :    600
Dynamics   Timestep [sec]   :    600
Emission   Timestep [sec]   :   1200
Unit Conv  Timestep [sec]   :    600
Diagnostic Timestep [sec]   :      0
Radiation  Timestep [sec]   :      0
 GEOS-Chem radiation time step (for RRTMG) does not
 agree with time step set in GCHP.rc
 GEOS-Chem RRTMG time step                     :    0.00000000    
 RRTMG_DT in GCHP.rc                           :    10800.0000    
```
Apparently the value of RRTMG_DT in `GCHP.rc` isn't getting copied to `Input_Opt%TS_RAD`, and that is the source of the error.

### Possible cause
It looks like tsRad is not getting passed to `GCHP_Chunk_Init` from `Chem_GridCompMod`:
```F90
  SUBROUTINE GCHP_Chunk_Init( nymdB,         nhmsB,      nymdE,           &
                              nhmsE,         tsChem,     tsDyn,           &
                              lonCtr,        latCtr,                      &
#if !defined( MODEL_GEOS )
                              GC,            EXPORT,                      &
#endif
                              Input_Opt,     State_Chm,  State_Diag,      &
                              State_Grid,    State_Met,  HcoConfig,       &
                              HistoryConfig, RC )
```

**Comment by yantosca** [comment]
Tagging @lizziel @Jourdan-He @SaptSinha

**Comment by msulprizio** [comment]
The PR for this has been merged into the `dev/14.1.0` branch in https://github.com/geoschem/geos-chem.
